### PR TITLE
Mention that `Sec-CH-UA-Mobile` is also a low-entropy hint

### LINKED ISSUE #117: Sec-CH-UA header field mentions inclusion in low-entropy table but Sec-UA-Mobile does not
This should be updated so that Sec-UA-Mobile also mentions its inclusion in the low-entropy table.

See: https://wicg.github.io/ua-client-hints/#sec-ch-ua
and
https://wicg.github.io/ua-client-hints/#sec-ch-mobile

Compared to https://wicg.github.io/client-hints-infrastructure/#low-entropy-table

### PR DESCRIPTION
**Pull request by amtunlimited**
Fixes #117

***
<a href="https://pr-preview.s3.amazonaws.com/amtunlimited/ua-client-hints/pull/122.html" title="Last updated on Aug 12, 2020, 3:07 PM UTC (9b32455)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/ua-client-hints/122/601c5ad...amtunlimited:9b32455.html" title="Last updated on Aug 12, 2020, 3:07 PM UTC (9b32455)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -347,6 +347,10 @@ The header's ABNF is:
   Sec-CH-UA-Mobile = sh-boolean
 ```
 
+Like the `Sec-CH-UA` hint above ([[#sec-ch-ua]]), since it's included in the <a
+href="https://wicg.github.io/client-hints-infrastructure/#low-entropy-table">low-entropy table</a>,
+the `Sec-CH-UA-Mobile` header will also be sent with all requests.
```
Comment (yoavweiss) [inline index.bs:352]: s/sent with all requests/sent by default/?
Feature/Permissions Policy can still make sure it's not sent.

Also, should this be a note?

Comment (amtunlimited) [reply index.bs:352]: That'd probably be good. There's a similar section in Sec-CH-UA, should I make that a note as well?

Comment (yoavweiss) [reply index.bs:352]: Yeah, assuming it doesn't define anything

Comment (amtunlimited) [reply index.bs]: Done

PTAL

File: index.bs [hunk 2]
```diff
@@ -347,6 +349,16 @@ The header's ABNF is:
...
 
+Note: Like `Sec-CH-UA` above, since it's included in the <a
+href="https://wicg.github.io/client-hints-infrastructure/#low-entropy-table">low-entropy table</a>,
+the `Sec-CH-UA-Mobile` header will be sent by default, whether or not the server opted-into
+receiving the header via an `Accept-CH` header (although it can still be controlled by it's
+<a href="https://wicg.github.io/client-hints-infrastructure/#policy-controlled-features">policy-controlled feature</a>.
+It is considered low enropy because it is a single bit of information directly controllable
+by the user.
+
+Note: These client hints can be evoked with the following set of <a href="https://wicg.github.io/client-hints-infrastructure/#client-hints-token-definition">Client Hint tokens</a>: UA-Arch, UA-Model, UA-Platform, UA, UA-Mobile
```
[the remark this answers is not in the document]
Comment (amtunlimited) [reply index.bs:360]: Borked the rebase order merging my other PR. The force should fix it

File: index.bs (hunk 1 above)
Comment (amtunlimited) [reply index.bs]: Done, PTAL

File: index.bs (hunk 2 above)
[the remark this answers is not in the document]
Comment (amtunlimited) [reply index.bs:360]: Done. PTAL

[the remark this answers is not in the document]
Comment (yoavweiss) [reply index.bs:360]: > Borked the rebase order merging my other PR. The force should fix it

computers :shrug:

**Pull request merged**